neo/test/coretest/test_irregularysampledsignal.py

```diff
@@ -550,6 +562,11 @@ def test_as_quantity(self):
         self.assertIsInstance(sig_as_q, pq.Quantity)
         assert_array_equal(self.data1, sig_as_q.magnitude.flat)
 
+    def test__copy_should_preserve_parent_objects(self):
```

Comment:
in test_analogsignals, this test has a different name (test__copy_should_let_access_to_parents_objects). Maybe adjust one of the two for consistency?